mandos.md

```diff
@@ -456,12 +500,18 @@ TODO make sure that none of the state changes are persisted -- [Doc](https://doc
 ```k
     syntax Step ::= transfer(TransferTx) [klabel(transfer), symbol]
  // -----------------------------------------------------
-    rule <k> transfer(TX) => TX ... </k> [priority(60)]
+    rule <k> transfer(TX) => TX ... </k>
+         <commands> . </commands>
+      [priority(60)]
 
     syntax TransferTx ::= transferTx    ( from: Address, to: Bytes, value: Int ) [klabel(transferTx), symbol]
                         | transferTxAux ( from: Bytes, to: Bytes, value: Int )   [klabel(transferTxAux), symbol]
  // ------------------------------------------------------------------------------------------------------------
-    rule <k> transferTx(FROM, TO, VAL) => transferTxAux(#address2Bytes(FROM), #address2Bytes(TO), VAL) ... </k> [priority(60)]
+    rule <k> transferTx(FROM, TO, VAL) 
+          => transferTxAux(#address2Bytes(FROM), #address2Bytes(TO), VAL) ...
+         </k>
+         <commands> . </commands>
+    [priority(60)]
```

Comment:
Is the priority still needed after you added an empty `commands` cell to the rule? (it may be, I don't know/remember why it was useful before)

Reply:
I think it is not necessary since we have `#wait` and `#endWasm` to switch between `k`, `commands`, and `instrs`, but I'm not sure either.